.github/workflows/translations-reporter.yml

```diff
@@ -0,0 +1,36 @@
+name: "Translations"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ main ]
```

Comment:
can you add this? so it will cancel ongoing runs in case a new one has been triggered - https://github.com/DeFiCh/wallet/blob/824158da6be57e0e9cc0697f2c1fefe5fe2c49e9/.github/workflows/pr-lint.yml#L9

Reply:
Updated